### PR TITLE
Feature/vao support dev 7320

### PR DESCRIPTION
**Pull request by Kuranes**
Adds support for Vertex Array object (on gl bind draw call for all vertex buffer binding and type)

### CODE REVIEW

**Comment by Kuranes** [comment]
- squashed commits for cleaner merge

File: sources/osg/DrawElements.js [hunk 1]
```diff
@@ -39,6 +39,12 @@ DrawElements.prototype = {
         var gl = state.getGraphicContext();
         gl.drawElements( this.mode, this.count, this.uType, this.offset );
     },
+    drawElements: function ( state, glParam ) {
```
Comment (cedricpinson) [inline sources/osg/DrawElements.js]: This part is not used we can remove the code

**Comment by Kuranes** [comment]
- removed no more necessary code

**Comment by cedricpinson** [comment]
refactored in another PR https://github.com/cedricpinson/osgjs/pull/570

**Pull request closed without merging**